lib/cli/commands/config.js

```diff
@@ -0,0 +1,77 @@
+import path from "node:path";
+import os from "node:os";
+import chalk from "chalk";
+import baseMiddleware from "../middlewares/base.js";
+// Internal module that is not exposed in package.json. Workaround in order to make it available for the CLI.
+import Configuration from "../../../node_modules/@ui5/project/lib/config/Configuration.js";
```

Comment:
Configuration is now available as package-export. See `config/Configuration`

Reply:
Thanks!